docs/source/support/release-notes.rst

```diff
@@ -6,6 +6,25 @@ Release notes
 
 This page contains release notes for the SDK.
 
+0.12.0
```

Comment:
Have we cut this release already? I thought the version number only goes here right before the SDK release is made. And everybody else just piles changes into the "unnamed" bucket?

Reply:
Yeah, I'm jumping the gun on item 1. in the new `Readme.md` here.